crypto/aes.go

```diff
@@ -0,0 +1,154 @@
+package crypto
+
+import (
+	"bufio"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/d1str0/pkcs7"
+	"github.com/golang/glog"
+)
+
+func LoadPrivateKey(privateKeyBase64 string) (*rsa.PrivateKey, error) {
```

Comment:
This function (and any other rsa helpers, which might be useful) should probably go on a separate `rsa.go` file. Or we can rename this file to just `decryption.go` or sth